test/cmd/create.sh

```diff
@@ -85,7 +85,9 @@ run_kubectl_create_error_tests() {
   rm "${ERROR_FILE}"
 
   # Posting a pod to namespaces should fail.  Also tests --raw forcing the post location
-  grep -q "cannot be handled as a Namespace: converting (v1.Pod)" <<< "$( kubectl convert -f test/fixtures/doc-yaml/admin/limitrange/valid-pod.yaml -o json | kubectl create "${kube_flags[@]}" --raw /api/v1/namespaces -f - --v=8 2>&1 )"
+  kubectl apply "${kube_flags[@]}" -f test/fixtures/doc-yaml/admin/limitrange/valid-pod.yaml
+  grep -q "cannot be handled as a Namespace: converting (v1.Pod)" <<< "$( kubectl get "${kube_flags[@]}" pod/valid-pod -o json | kubectl create "${kube_flags[@]}" --raw /api/v1/namespaces -f - --v=8 2>&1 )"
```

Comment:
No need for double quotes on static strings. Suggestion:
```shell
grep -q 'cannot be handled as a Namespace: converting (v1.Pod)' <<< "$( kubectl get "${kube_flags[@]}" pod/valid-pod -o json | kubectl create "${kube_flags[@]}" --raw /api/v1/namespaces -f - --v=8 2>&1 )"
```

Reply:
Why not just do:
```bash
kubectl create "${kube_flags[@]}" -f test/fixtures/doc-yaml/admin/limitrange/valid-pod.yaml --raw /api/v1/namespaces --v=8
```
It will give you the exact same effect. Alternatively `kubectl-convert` if you want to stick with convert being used.